sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -1361,30 +1361,35 @@ public void Cfg_Coalesce()
 
             VerifyAllInstructions(branchBlock, "b");
             VerifyAllInstructions(bNullBlock, "c");
-            VerifyAllInstructions(afterOp, "a = b ?? c");
+            VerifyAllInstructions(afterOp, "b", "a = b ?? c");
         }
 
         [TestMethod]
         [TestCategory("CFG")]
         public void Cfg_Coalesce_Multiple()
         {
             var cfg = Build("var a = b ?? c ?? d;");
-            VerifyCfg(cfg, 5);
+            VerifyCfg(cfg, 6);
 
             var bBlock = cfg.EntryBlock as BinaryBranchBlock;
             var blocks = cfg.Blocks.ToList();
             var cBlock = blocks[1] as BinaryBranchBlock;
-            var dBlock = blocks[2];
-            var bcdBlock = blocks[3];   // b ?? c ?? d
+            var cResultBlock = blocks[2];
+            var dBlock = blocks[3];
+            var bcdBlock = blocks[4];   // b ?? c ?? d
             var exitBlock = cfg.ExitBlock;
 
             bBlock.SuccessorBlocks.Should().OnlyContainInOrder(cBlock, bcdBlock);
-            cBlock.SuccessorBlocks.Should().OnlyContainInOrder(dBlock, bcdBlock);
+            cBlock.SuccessorBlocks.Should().OnlyContainInOrder(dBlock, cResultBlock);
```

Comment:
same here, if you could check the `TrueSuccessor` and `FalseSuccessor` instead of `SuccessorBlocks`